google/cloud/security/common/gcp_api/bigquery.py

```diff
@@ -59,19 +59,22 @@ def get_bigquery_projectids(self):
             If there are no project_ids enabled for bigquery an empty list will
             be returned.
         """
+        key='projects'
```

Comment:
I would make this a karg and default it to 'projects'. Otherwise this becomes a magic string that a caller can't influence if things change in how the data is key'd.